GitHub_Learning/Feature/News/View/NewsViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NewsViewController.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 05.01.2021.
+//  Copyright © 2021 Amin_Second_Test_Project. All rights reserved.
+//
+
+import UIKit
+
+class NewsViewController: UIViewController {
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        tabBarItem = UITabBarItem(tabBarSystemItem: .bookmarks, tag: 2)
```

Comment:
Ty tagy tam k nicemu nemas. Nemusis je pouzivat.

Reply:
Bohužel tu není žádnej jinej init na systemTabBarItem, a nil to být nemůže. Nechtělo se mi hledat ikony a dávat si tam vlastní tak jsem si chtěl usnadnit práci tim co je od systemu.

Reply:
Nejde to tam proste vynechat? Nekdy je tam default value, ktera nemusi byt hned videt.

Reply:
Když vynechám tak to řve. Tak nevim, nebo nevim jak no.